paper.md

```diff
@@ -24,8 +24,8 @@ bibliography: paper.bib
 
 # Summary
 
-The tensor framework JAX [@jax] combines expressivity and performance while providing an accessible pure Python interface.
-In particular, JAX is expressive due to its clean, functional design, and performant due to a powerful JIT (just-in-time) compiler.
+The tensor framework JAX [@jax] combines expressivity and performance while retaining an accessible pure Python interface.
+Expressivity is achieved by treating functions as first-class objects, while efficiency is obtained by compiling to machine code Just-Ahead-Of-Time.
```

Comment:
Any particular reason why you insist on Just-Ahead-Of-Time? JAX itself calls it just-in-time:

> JAX is Autograd and XLA, brought together for high-performance numerical computing and machine learning research. It provides composable transformations of Python+NumPy programs: differentiate, vectorize, parallelize, Just-In-Time compile to GPU/TPU, and more.

Reply:
It's probably jargon i got from meddling in the julia world, though I heard it when chatting with a few Computer scientists too..

Traditionally JIT languages are compiled during runtime and executed inside a Virtraul Machine.
Like Java, that executes the code in the VM until a function is hit ~30 times, at which points he compiles it and subsequent invocations will run the machine code instead of in the VM.

Jax (like julia) does not do that: it's not even a dynamic language (it just fakes it).
jax compiles code like gcc would, generates an ASM blob that is fed to the relevant device.
This is AOT, not JIT.

It's just that the compilation is postponed to the last possible moment: when you need the result.

Reply:
Apparently according to wiipedia

> Julia uses a just-in-time (JIT) compiler that is referred to as "just-ahead-of-time" (JAOT) in the Julia community, as Julia compiles all code (by default) to machine code before running it.[23][24]

I'm okay if you want to revert it.
But i stand by my ideas.